packages/suite/src/utils/wallet/coinmarket/__tests__/coinmarketUtils.test.ts

```diff
@@ -81,4 +82,12 @@ describe('coinmarket utils', () => {
             },
         ]);
     });
+
+    it('getTagAndInfoNote', () => {
+        expect(getTagAndInfoNote({})).toStrictEqual({ infoNote: undefined });
+        expect(getTagAndInfoNote({ infoNote: '' })).toStrictEqual({ infoNote: '' });
+        expect(getTagAndInfoNote({ infoNote: 'Foo' })).toStrictEqual({ infoNote: 'Foo' });
```

Comment:
nit: consider adding one more test case to explicitly show that `#` matters only at the beginning.

Are the trailing spaces handled for infoNote somewhere? What about `infoNote: ' #Foo'`?

```suggestion
        expect(getTagAndInfoNote({ infoNote: 'Foo' })).toStrictEqual({ infoNote: 'Foo' });
        expect(getTagAndInfoNote({ infoNote: 'Foo#Bar' })).toStrictEqual({ infoNote: 'Foo#Bar' });
```